templates/stats.ejs

```diff
@@ -1,324 +1,297 @@
+            if (document.getElementById(stat_value_span_id) === null) {continue;}
+            document.getElementById(stat_value_span_id).innerHTML = stat_value;
+            document.getElementById(stat_field_id).className = "stat-field" + 
+                (stat === "v_available_points" ? " available" : "") +
+                (stat === "v_magical_power" || stat === "v_magical_knowledge" ? " magic-special" : "");
+        }
+    }
 
-            var stats_aux = getAuxiliaryStats();
-            initializeStats(stats_aux);
-            var stats_changed = false; // This variable is used to check if the stats have been changed
+    var stats_aux = getAuxiliaryStats();
+    initializeStats(stats_aux);
+    var stats_changed = false; // This variable is used to check if the stats have been changed
 
-            function updateStat(stat, stat_aux_key, stat_field_id, stat_field_value_id) {
-                var stat_aux_value = stats_aux[stat_aux_key];
-                var stat_max = locals["v_max_stat"];
-                var available_points = stats_aux["v_available_points_aux"];
+    function updateStat(stat, stat_aux_key, stat_field_id, stat_field_value_id) {
+        var stat_aux_value = stats_aux[stat_aux_key];
+        var stat_max = locals["v_max_stat"];
+        var available_points = stats_aux["v_available_points_aux"];
 
-                if ((stat_aux_value >= stat_max) || (available_points <= 0)) { return; }
-                stats_aux[stat_aux_key] += 1;
-                stats_aux["v_available_points_aux"] -= 1;
-                document.getElementById("available_points_value").innerHTML = stats_aux["v_available_points_aux"];
-                document.getElementById(stat_field_value_id).innerHTML = stats_aux[stat_aux_key];
-                document.getElementById(stat_field_id).className = "stat-field updated";
-                stats_changed = true;
-            }
+        if ((stat_aux_value >= stat_max) || (available_points <= 0)) { return; }
+        stats_aux[stat_aux_key] += 1;
+        stats_aux["v_available_points_aux"] -= 1;
+        document.getElementById("available_points_value").innerHTML = stats_aux["v_available_points_aux"];
+        document.getElementById(stat_field_value_id).innerHTML = stats_aux[stat_aux_key];
+        document.getElementById(stat_field_id).className = "stat-field updated";
+        stats_changed = true;
+    }
 
-            function confirmStats() {
-                if (!stats_changed) { return; }
-                // Update the stats cookies
-                for (var stat_aux in stats_aux) {
-                    var stat = stat_aux.split("_aux")[0];
-                    document.cookie = stat + "=" + stats_aux[stat_aux];
-                }
-                // Refresh the page to reflect the changes
-                window.location.reload();
-            }
-        </script>
-      </head>
-      <body>
-        <main id="main">
-            <div class="header" style="justify-content: center;">
```

Comment:
The missing header would mean that we would see the same header as for the main page.
I can fix that later, maybe by conditional headers.

Reply:
or we could accept this as a new feature, where headers will be consistent throughout.
(But that would make the title header weird, as the Book and Chapter name may not be applicable for a specific page, so we can conditionally hide that depending on where we are.)

Reply:
Good catch; I added something that changes the header to "Stats" or "Menu" when you go on different pages.